src/groups.jl

```diff
@@ -4,7 +4,15 @@
 
 immutable BenchmarkGroup
     tags::Vector{Any}
-    data::Dict{Any,Any}
+    data::Associative
+
+    function BenchmarkGroup(tags, data::Dict)
+        new(convert(Vector{Any}, tags), convert(Dict{Any, Any}, data))
+    end
+
+    function BenchmarkGroup(tags, data)
+        new(convert(Vector{Any}, tags), data)
+    end
 end
```

Comment:
If we're allowing `data::Associative`, might as well not have these inner constructors.